src/Controller/Admin/ConfigController.php

```diff
@@ -39,4 +42,30 @@ public function edit(Request $request): Response
             'form' => $form->createView(),
         ]);
     }
+
+    /**
+     * @Route("/admin/config/telemetry", name="admin_config_telemetry_enable", methods={"POST"})
+     */
+    public function enableTelemetry(Request $request): Response
+    {
+        $this->telemetry->generateInstanceId();
+        $this->dispatchMessage(new ChangeConfig([
+            'telemetry' => 'enable',
+        ]));
+
+        return $this->redirectToRoute('index');
+    }
+
+    /**
+     * @Route("/admin/config/telemetry", name="admin_config_telemetry_disable", methods={"DELETE"})
+     */
+    public function disableTelemetry(Request $request): Response
+    {
+        $this->telemetry->generateInstanceId();
+        $this->dispatchMessage(new ChangeConfig([
+            'telemetry' => 'disable',
+        ]));
```

Comment:
maybe it can be packed into one action:
```php
// @Route("/admin/config/telemetry", name="admin_config_telemetry", methods={"POST", "DELETE"})

$this->dispatchMessage(new ChangeConfig([
    'telemetry' => $request->isMethod(Request::METHOD_POST) ? 'enable' : 'disable'
]));
```

Reply:
I've changed that, although it's not marked as "outdated"